axlearn/cloud/gcp/dataflow_inference_custom.py

```diff
@@ -0,0 +1,205 @@
+"""An Apache Beam example pipeline to run batch inference jobs using a model trained with AXLearn.
```

Comment:
```suggestion
"""An Apache Beam example pipeline to run batch inference jobs using a model trained with AXLearn.

```

Can we also make a separate directory like `axlearn/cloud/gcp/examples` with these scripts, to distinguish from libraries?